core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -759,6 +761,14 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
           }, function() {
             $scope.isSaveInProgress = false;
           });
+
+          if ($scope.isPrivate()) {
+            siteAnalyticsService.registerSaveExplorationDraftEvent(
```

Comment:
Yes. I feel that giving them divorced names _is_ a tight coupling to private/public status, as then you're regarding the two types of exploration as having two different operations possible (saving and publishing respectively). When it's really the same operation. There's also a risk of confusion between 'publish exploration' and 'publish changes'.